### PR TITLE
Drag n zoom

### PR DESCRIPTION
**Pull request by simontegg**
Problem: sometimes part of the chart is drawn off-screen.

This pr allows user to drag the graph by holding down the mouse and moving and zoom around a point by using the mousewheel or two finger zoom on the touchpad.
- clicking on an agent no the context brings that agent back to the centre.
- dragging and zooming update the querystring, allowing the user to share particular views.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,6 +97,7 @@
     "lodash.reject": "^3.1.1",
     "lokijs": "^1.2.3",
     "micro-req": "^1.0.0",
+    "matrix-transform": "holodex/matrix-transform",
```
Comment (ahdinosaur) [inline package.json]: use [matrix-zoom](https://www.npmjs.com/package/matrix-zoom)

File: .gitignore [hunk 2]
```diff
@@ -6,6 +6,7 @@
 *.swp
 
 /src/assets/styles.css
+/src/assets/bundle.js
```
Comment (ahdinosaur) [inline .gitignore]: rm this, also any `src/assets/bundle.js` you still have around.

**Comment by ahdinosaur** [comment]
this looks really great! :deciduous_tree:

**Comment by simontegg** [comment]
Ta,

I've noticed a bug:   when tags are removed it sometime resets the vector back to a previous position if you'v been dragging it around. Probably related to the mutable vector state split over `colaStore` and `routeStore`

**Comment by ahdinosaur** [comment]
ah hmm okay, happy to merge a bug fix, let me know if you want some :eyes: anywhere.

**Comment by ahdinosaur** [comment]
[deployed](http://holodex.enspiral.com/) :tada:

**Pull request merged**